corelibs/libvirtio.md

```diff
@@ -1,9 +1,277 @@
 # VirtIO library (libvirtio)
 
-Documentation under development.
+`libvirtio` provides an abstraction layer for working with VirtIO devices. VirtIO is a standard for network and disk
+device drivers where just the guest's device driver "knows" it is running in a virtual environment, and cooperates
+with the hypervisor.
```

Comment:
Let's not anthropomorphise our system that easily ;) My suggestion is just a suggestion, perhaps it might be written better.
```suggestion
`libvirtio` provides an abstraction layer for working with VirtIO devices. VirtIO is a standard dedicated to provide
a common device API (e.g. network and mass storage adapters) for virtualized execution environment.
```